Userland/Libraries/LibWeb/DOM/Event.idl

```diff
@@ -19,21 +19,20 @@ interface Event {
     readonly attribute unsigned short eventPhase;
 
     undefined stopPropagation();
-    attribute boolean cancelBubble;
+    attribute boolean cancelBubble; // legacy alias of .stopPropagation()
     undefined stopImmediatePropagation();
 
     readonly attribute boolean bubbles;
     readonly attribute boolean cancelable;
-    attribute boolean returnValue;
+    attribute boolean returnValue;  // legacy
     undefined preventDefault();
     readonly attribute boolean defaultPrevented;
     readonly attribute boolean composed;
 
-    readonly attribute boolean isTrusted;
-    readonly attribute double timeStamp;
-
-    undefined initEvent(DOMString type, optional boolean bubbles = false, optional boolean cancelable = false);
+    [LegacyUnforgeable] readonly attribute boolean isTrusted;
+    readonly attribute DOMHighResTimeStamp timeStamp;
```

Comment:
`Event::time_stamp()` should probably be updated to return `DOMHighResTimeStamp` as well, for clarity.

Reply:
Absolutely!